### DISCUSSION TITLE
articles/article/2024/11/202411170

### DISCUSSION THREAD

**Discussion by giscus** [yyyytawa/yyyytawa.github.io#11] [Announcements]
# articles/article/2024/11/202411170

本文写于 20241117-20241117 序言 这个文章只是说一些事情.希望老师不要公开处刑 如果要是打不开的话就用https,由于服务器在海外的Cloudflare Pages上所以大陆访问可能会有点慢(容易吃GFW的阻断)(虽然说已经配置了PWA来改善弱网/断网条件下的访问,但是无法保证一定能打开) 对父母的一些话 (一个小时不到的的赶工产品,...

https://docs.yyyyt.top/articles/article/2024/11/202411170.html

**Comment by runoneall** [comment]
我是评论

**Reply by yyyytawa** [reply]
我也是评论
